web/packages/teleterm/src/ui/services/workspacesService/accessRequestsService.ts

```diff
@@ -433,6 +434,18 @@ export function toResourceRequest({
         },
         kind: 'app',
       };
+    case 'aws_ic_account_assignment':
+      return {
+        resource: {
+          uri: routing.getAppUri({
+            rootClusterId,
+            leafClusterId,
+            appId: resourceId,
+          }),
+          samlApp: false,
+        },
+        kind: 'app',
+      };
```

Comment:
Does it work in Connect at the moment at all? I don't have a cluster I could test this with, so I've changed this to always return `true`:

https://github.com/gravitational/teleport/blob/f8bfc867c62616a74bbfa12e98819aaefca271b5/web/packages/teleterm/src/ui/DocumentCluster/UnifiedResources.tsx#L198

…and everything seems to work. But I'm doing this on Ben's cluster where I can't make access requests and I don't have privileges to change this, so I couldn't test the whole feature. ;f

Reply:
It does not work in Connect right now. I only added these cases in teleport package cause the typescript was complaining otherwise. I am looking into Connect separately though.